### PR TITLE
DOCS-3451 SCALE 22.02 Reorg Permissions Article Content

### PR DESCRIPTION
**Pull request by micjohnson777**
This PR creates a new UI reference article for ACL editor screens, reorganizes the content in the how-to article, and updates screenshots.

Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+| **Apply User** | Select to confirm user changes. To prevent errors, TrueNAS only submits changes only after you select this option. |
+| **Group** | Enter or select the group to control the dataset. Groups created manually or imported from a directory service appear in the menu. |
+| **Apply Group** | Select to confirm group changes. To prevent errors, TrueNAS only submits changes only after you select this option. |
+{{< /expand >}}
+### Access Settings
+The **Access** section lets users define the basic **Read**, **Write**, and **Execute** permissions for the **User**, **Group**, and **Other** accounts that might access this dataset.
+
+{{< hint warning >}}
+A common misconfiguration is removing the **Execute** permission from a dataset that is a parent to other child datasets.
+Removing this permission results lost access to the path.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md:50]:
```suggestion
Removing this permission results in lost access to the path.
```

File: content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+| **Select a preset ACL** | Click this radio button to populate the **Default ACL Options** dropdown list with a set of pre-configured POSIX permissions. |
+| **Create a custom ACL** | Click this radio button to display the **Edit ACL** screen with no default permissions, users or groups to configure your own set of permissions after you click **Continue**. |
+
+Click **Continue** to display the **Edit ACL** screen.
+{{< /expand >}}
+## Edit ACL Screen
+
+The **Edit ACL** screen displays different setting options based on the **ACL Type** setting on the **Add Dataset** or **Edit Dataset** screen in the **Advanced Options** section.
+
+The section below describe the differences between screens for each ACL type.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md:89]:
```suggestion
The section below describes the differences between screens for each ACL type.
```

File: content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+### ACL Editor Settings - POSIX and NFSv4
+
+Select any user account or group manually entered or imported from a directory service in the **Owner** or **Owner Group**. The value entered or selected in each field displays in the **Access Control List** below these fields. 
+
+**Dataset** displays the dataset path (name) you selected to edit.
+
+![ACLEditorSettings](/images/SCALE/22.02/ACLEditorSettings.png "ACL Editor Settings") 
+
+### Access Control List - POSIX and NFS4
+The **Access Control List** section displays the items and a permissions summary for the **owner@**, **group@** and **everyone@** for both POSIX and NSFv4 ACL types. The list of items changes based on a selected pre-configured set of permissions.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md:100]:
```suggestion
The **Access Control List** section displays the items and a permissions summary for the **owner@**, **group@**, and **everyone@** for both POSIX and NSFv4 ACL types. The list of items changes based on a selected pre-configured set of permissions.
```

File: content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+To add a new item to the ACL, click **Add Item**, define **Who** the Access Control Entry (ACE) applies to, and configure permissions and inheritance flags for the ACE.
+
+### Edit ACL Functions - POSIX and NFS4
+These functions display on the **Edit ACL** screen for both POSIX and NSFv4 ACL types except for **Strip ACL**, it only displays for NSFv4 types.
```
Comment (tonyriv3) [inline content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md:105]:
```suggestion
These functions display on the **Edit ACL** screen for both POSIX and NSFv4 ACL types except for **Strip ACL**, which only displays for NSFv4 types.
```

**Review by tonyriv3 — CHANGES_REQUESTED** [review]

File: content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md [hunk 5]
```diff
@@ -0,0 +1,206 @@
...
+ - scaleacls
+ - scaledatasets
+ - scalestorage
+---
+
+
+{{< toc >}}
+
+
+TrueNAS SCALE offers two ACL types, POSIX which is the SCALE default, and NFSv4.
```
[the remark this answers is not in the document]
Comment (micjohnson777) [reply content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md]: It is not allowing me to commit your suggestion for some reason. After Tim reviews, I'll make this change if it is not already in the article now.

[the remark this answers is not in the document]
Comment (Mrt134) [reply content/SCALE/SCALEUIReference/Storage/Pools/EditACLScreens.md]: This change has already been made and is good to go

**Review by tonyriv3 — APPROVED** [review]

**Pull request merged**